Content.Server/Disposal/Unit/Components/DisposalUnitComponent.cs

```diff
@@ -65,5 +65,13 @@ public sealed class DisposalUnitComponent : SharedDisposalUnitComponent, IGasMix
 
         [DataField("air")]
         public GasMixture Air { get; set; } = new(Atmospherics.CellVolume);
+
+        [DataField("flushTime")]
+        public float FlushTime = 2.0f;
+
+        /// <summary>
+        /// Moment the flush started to calculate end of animation
+        /// </summary>
+        public TimeSpan? FlushStart;
```

Comment:
Alright so I playtested ingame but not sure what this is supposed to do? The visualstate doesn't change after 2 seconds and the state updates at the end when it's repressurised.

Reply:
Ok, thought it might be my submodules out of date, but still seems to work for me after updating them and merging upstream into the PR branch.